ui/appui-abstract/src/appui-abstract/utils/IconSpecUtilities.ts

```diff
@@ -6,8 +6,10 @@
  * @module Utilities
  */
 
+/* eslint-disable deprecation/deprecation */
+
 /** Creates an IconSpec with an SVG source and gets the SVG source from an IconSpec.
- * @public
+ * @public @deprecated in 4.3. AppUI libraries > 4.6.x support loading SVGs sources without prefixes, eliminating the need for these utilities.
```

Comment:
Will be at least 4.7.x

Reply:
It should be changed in the 2 other deprecations too :)

Reply:
That's a forehead slapper. Thanks. Fixed.